tests/unit/test_distro.py

```diff
@@ -312,28 +292,8 @@ def test__configure_metrics_export_env_defaults_valid_protocol_grpc(self, mocker
         assert os.environ.get(OTEL_EXPORTER_OTLP_METRICS_ENDPOINT) == "https://otel.collector.na-01.cloud.solarwinds.com:443/v1/metrics"
         assert os.environ.get(OTEL_EXPORTER_OTLP_METRICS_HEADERS) == "authorization=Bearer%20foo-token"
 
-    def test__configure_traces_export_env_defaults_none_protocol(self, mocker):
```

Comment:
This test removed because protocol check + setting as default `http/protobuf` are now done by `_configure`.